framework/src/main/java/org/apache/drill/test/framework/CmdParam.java

```diff
@@ -52,6 +52,9 @@
     @Parameter(names = {"-r", "--report"}, description = "Generate json report", required=false)
     public boolean generateReports = false;
 
+    @Parameter(names = {"-z", "--negativeCases"}, description = "Include fail test cases", required=false)
```

Comment:
Change name to "-if" --"includeFailed"
Change description to "Include tests tagged as failures"